changelogs/fragments/2040-zos_backup_restore-fixed_return_backup_name

```diff
@@ -0,0 +1,6 @@
+trivial:
+   - zos_backup_restore - Returned the backup_name and added assertion in test suite
```

Comment:
Please follow the `bugfixes` entry format from https://docs.ansible.com/ansible/latest/community/development_process.html#changelog-fragment-entry-format

```
bugfixes
Fixes that resolve issues. SHOULD not be used for minor enhancements (use minor_change instead). Use past tense to describe the problem and present tense to describe the fix.
```

That would result in:
```suggestion
   - zos_backup_restore - Return value ``backup_name`` was empty upon successful result. Fix now returns ``backup_name`` populated.
```